Backend/src/main/java/idealab/api/model/PrintJob.java

```diff
@@ -18,14 +22,17 @@
     private Integer id;
 
     @ManyToOne()
+    @Audited(targetAuditMode = NOT_AUDITED)
     @JoinColumn(name="fk_customer_info_id", referencedColumnName = "id", nullable = false)
     private CustomerInfo customerInfo;
 
     @ManyToOne()
+    @Audited(targetAuditMode = NOT_AUDITED)
     @JoinColumn(name="fk_color_type_id", referencedColumnName = "id", nullable = false)
     private ColorType colorTypeId;
 
     @ManyToOne()
+    @Audited(targetAuditMode = NOT_AUDITED)
     @JoinColumn(name="fk_employee_id", referencedColumnName = "id", nullable = false)
     private Employee employeeId;
 
```

Comment:
Is it okay if we don't audit queue ID?

Reply:
I added `@NotAudited` to the queue ID so it shouldn't show up in the table anymore.